cmd/kube-proxy/app/server_windows.go

```diff
@@ -81,8 +81,7 @@ func (s *ProxyServer) platformCheckSupported() (ipv4Supported, ipv6Supported, du
 // createProxier creates the proxy.Provider
 func (s *ProxyServer) createProxier(config *proxyconfigapi.KubeProxyConfiguration, dualStackMode, initOnly bool) (proxy.Provider, error) {
 	if initOnly {
-		// --init-only is not implemented on Windows
-		return nil, nil
+		return nil, fmt.Errorf("Not supported on Windows: --init-only")
```

Comment:
errors should start with a lowercase letter, so "not", but I think the original comment text ("--init-only is not implemented on Windows") is a better message here anyway...